runtime/parachains/src/paras/tests.rs

```diff
@@ -1252,8 +1252,21 @@ fn pvf_check_upgrade_reject() {
 		Paras::schedule_code_upgrade(a, new_code.clone(), RELAY_PARENT, &Configuration::config());
 		check_code_is_stored(&new_code);
 
-		// Supermajority of validators vote against `new_code`. PVF should be rejected.
-		IntoIterator::into_iter([0, 1, 2, 3])
+		// 1/3 of validators vote against `new_code`. PVF should not be rejected yet.
+		IntoIterator::into_iter([0])
+			.map(|i| PvfCheckStatement {
+				accept: false,
+				subject: new_code.hash(),
+				session_index: EXPECTED_SESSION,
+				validator_index: i.into(),
+			})
+			.for_each(sign_and_include_pvf_check_statement);
```

Comment:
Nit: looks like the loop has become singular and is not needed anymore?

```suggestion
		sign_and_include_pvf_check_statement(PvfCheckStatement {
			accept: false,
			subject: new_code.hash(),
			session_index: EXPECTED_SESSION,
			validator_index: 0.into(),
		});
```

Reply:
Good catch, thanks!